project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/templates/quick-post-form.tpl.php

```diff
@@ -4,6 +4,13 @@
  * @file
  * Template to render the Quick post forms.
  */
+
+global $user;
```

Comment:
@guytepper 
this kind of logic should never be in a template.
please move it to the preprocess function.